sentry-kubernetes.py

```diff
@@ -173,6 +174,9 @@ def watch_loop():
             fingerprint = []
             tags = {}
 
+            if CLUSTER_NAME:
+                tags["cluster"] = CLUSTER_NAME
```

Comment:
From what I understand, `cluster` is not some kind of special tag.

Wouldn't it be better to allow users to pass whatever tags they want to attach to events? Which then can be used to add this cluster tag, amongst others.

Not sure what the best way is to express that as a environment variable value though.

`ADDITIONAL_TAGS="cluster=europe-b foo=bar"`

Reply:
Arbitrary tags is a nice to have and is another way to solve the problem with more flexibility however I think minimum viable project says this PR is good enough.  It certainly solves the issue I have with multiple Sentry projects to track events from multiple K8 clusters.

Reply:
I agree that `ADDITIONAL_TAGS` will solve the same issue in better way but I wanted to keep it simple but as @maestrofjp said  most of us just want `cluster` tag

IMHO `ADDITIONAL_TAGS` is also good, but it will take some work to add this :-)

> Not sure what the best way is to express that as a environment variable value though.

also this is a big unknown when adding `ADDITIONAL_TAGS`, so I think let's keep it simple and [YAGNI](https://www.martinfowler.com/bliki/Yagni.html)